src/Symfony/Component/DependencyInjection/composer.json

```diff
@@ -21,7 +21,7 @@
     },
     "require-dev": {
         "symfony/yaml": "~3.4|~4.0",
-        "symfony/config": "~3.3|~4.0",
+        "symfony/config": "~3.4|~4.0",
```

Comment:
the conflict line should also be updated

Reply:
done.

Reply:
because method_exists? it could yes, let's do